docs/sdks/sdk-features/debugging-flag-assignment.md

```diff
@@ -54,6 +54,34 @@ getStringAssignment(flagKey, subjectKey, subjectAttributes, defaultValue);
 // => "control"
 ```
 
+### Details on Variation Keys for flag types
+
+Eppo will always log the value of `variationKey` in the Evaluation Details object. The `variationValue` represents the value created for that variation on the flag in it's orginal data type. Eppo logs the `variationKey` as a string to make it easier to send those logs to your data warehouse. Since String flags already log the variation value as a string, the `variationKey` and `variationValue` will be the same.
+
+For example a Boolean flag will have:
+
+```typescript
+variationKey: 'true',
+variationValue: true
+```
+
+and a Numeric flag will have:
+
+```typescript
+variationKey: '3',
+variationValue: 3
+```
+
+An exception exists for JSON flags where the `variationKey` will be the variation name given in the flag:
```

Comment:
Note: it's not exactly the name (lower cased and dashes added), so "My Variation" becomes "my-variation" -- and it doesn't change when the name gets updated. Not sure how much of this detail you want to add

Reply:
ah good catch -- i'll add those, i think it's important for customers to know